src/app/lorax/directives/modal-share.js

```diff
@@ -152,14 +153,21 @@ define(['angular', 'jquery'], function (angular, $) {
     };
 
     ModalShareController.prototype.shareFacebook = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/facebook/');
         window.open('https://www.facebook.com/sharer/sharer.php?u=' + issue.getShareUrl(), '_blank');
     };
 
     ModalShareController.prototype.shareTwitter = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/twitter/');
         window.open('http://twitter.com/share?text=' + issue.getTitle() + '&url=' +
             issue.getShareUrl() + '&hashtags=' + this._$scope.modalShare.content.hashtag, '_blank');
     };
 
+    ModalShareController.prototype.shareEmail = function (issue) {
+        ga('send', 'pageview', '/share' + issue.getUrl() + '/email/');
+        window.open('mailto:?subject=Shape of the Web - ' + issue.getTitle() + '&body=' + issue.getShareUrl());
```

Comment:
Thinking more, keeping "Shape of the Web" out of the subject would help deal with the fact that the structure of the subject (mainly the left-to-right nature of the stuff around the dash) needs to be part of the localization, so this whole thing needs some string formatting.

Also, you should URI encode the stuff you're passing to the subject and body.